base/acme/src/main/java/org/dogtagpki/acme/server/ACMEEngineConfigDefaultSource.java

```diff
@@ -15,8 +15,10 @@
  */
 class ACMEEngineConfigDefaultSource extends ACMEEngineConfigSource {
     Consumer<Boolean> setEnabled;
+    Consumer<Boolean> setWildcard;
```

Comment:
Not quite sure how this works. When are these fields going to be set/used?

Reply:
I dunno. I removed both and its not causing problems because its available from the parent class as well.